.github/actions/test-rust/action.yaml

```diff
@@ -46,17 +47,7 @@ runs:
     # have two different ones; we welcome a change to simplify this)
     - name: 📋 Test
       uses: richb-hanover/cargo@v1.1.0
-      if:
-        ${{ (inputs.target_option != '--target=wasm32-unknown-unknown' &&
-        runner.os == 'ubuntu') }}
+      if: ${{ (inputs.target_option == '' && runner.os == 'ubuntu') }}
       with:
         command: insta
         args: test --unreferenced=auto ${{ inputs.target_option }}
-    - name: 📋 Test
-      uses: richb-hanover/cargo@v1.1.0
-      if:
-        ${{ ! (inputs.target_option != '--target=wasm32-unknown-unknown' &&
-        runner.os == 'ubuntu') }}
-      with:
-        command: insta
-        args: test ${{ inputs.target_option }}
```

Comment:
This requires ` --unreferenced=auto` on wasm, to not delete a bunch of snapshots. We do this so we can test there are no unreferenced snapshots around (but not all tests run on wasm)

Reply:
I was having problems with wasm because insta does not have a nice way of excluding a crate like other cargo commands.

If I understand correctly, we already run `cargo test` for wasm and check for unreferenced snapshots in the first insta step?